style.css

```diff
@@ -0,0 +1,219 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+html {
+    font-size: 62.5%;
+}
+
+body {
+    font-family: 'Montserrat', sans-serif;
+    font-size: 1.6rem;
+    box-sizing: border-box
+}
+
+.wrapper {
+    margin: 0 auto;
+    width: 100%;
+    max-width: 1200px;
+}
+
+a:visited {
```

Comment:
If you change it to
```
a:hover, a:visited, a:link, a:active {
  /* ... */
}
```
you will be able to simplify your `.navbar-link` styles

Reply:
Done